src/libsync/networkjobs.cpp

```diff
@@ -430,10 +431,19 @@ void CheckServerJob::start()
     req.setAttribute(QNetworkRequest::RedirectPolicyAttribute, QNetworkRequest::NoLessSafeRedirectPolicy);
     req.setRawHeader(QByteArrayLiteral("OC-Connection-Validator"), QByteArrayLiteral("desktop"));
     req.setMaximumRedirectsAllowed(_maxRedirectsAllowed);
+
+    if (_clearCookies && Theme::instance()->connectionValidatorClearCookies()) {
```

Comment:
Since you're not running that check elsewhere, does it make sense here? Perhaps the variable should be called `_requestClearCookies` to signalize it won't necessarily clear the cookies.

Reply:
The feature is opt in and we don't want to mess with other providers.